common/src/main/java/dev/ftb/mods/ftbchunks/client/gui/WaypointEditorScreen.java

```diff
@@ -56,7 +57,7 @@ public WaypointEditorScreen() {
                 (widget, button) -> toggleAll(true));
     }
 
-    private static FormattedText ellipsize(Font font, FormattedText text, int maxWidth) {
+    public static FormattedText ellipsize(Font font, FormattedText text, int maxWidth) {
```

Comment:
Since we're updating Lib anyway for this, maybe this should be moved there :stuck_out_tongue: